docs/HowToGuides/GettingStarted.md

```diff
@@ -190,6 +190,10 @@ toolchain as a one-off, there are a couple of differences:
 > LLDB currently requires at least `swig-1.3.40` but will successfully build
 > with version 2 shipped with Ubuntu.
 
+> **Note**  
+> When running Linux in a VM, make sure to have at least 8 GB (when using flag `--release-debuginfo`) and 32 GB (when using flag `--debug-swift`) of RAM and 150 GB of disk space available. To speed up compilation of the toolchain, it is better to dedicate as many CPUs as possible on the host machine.
```

Comment:
"running Linux in a VM" -> "building on linux" as the same requirements outside VM
"8 GB (when using flag `--release-debuginfo`)" -> "8 GB of RAM when using flag `--release-debuginfo`"
"32 GB (when using flag `--debug-swift`) of RAM and 150 GB" -> "32 GB of RAM when using flag `--debug-swift`, plus 150 GB"

I think you can cut the last two sentences you added, starting with "To speed up."

Reply:
👋🏻  Hello @finagolfin ,

thank you very much for your input (and posts on Swift Forums!). Really essential, always.

Updated the text 👍🏻